app/views/wikis/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+table, th, td {
+    border: 1px solid blue;
+    border-collapse: collapse;
+}
+th, td {
+    padding: 5px;
+    width: 500px;
+}
+</style>
```

Comment:
Try to keep the CSS in the stylesheet files. This makes CSS easier to maintain as its more predictable where you will find it when you need to change it.

Consider moving the CSS in the view template to the stylesheet files.